admin-dev/themes/default/template/controllers/cart_rules/informations.tpl

```diff
@@ -87,7 +87,7 @@
 	</div>
 </div>
 
-<div class="form-group" id="cart-rules-highlight"{if !$currentTab->getFieldValue($currentObject, 'code')} style="display: none;"{/if}>
```

Comment:
@Hlavtox I think it does make sense not to highlight a code... if there's no code, doesn't it?

Reply:
No, my clients have “actions” like “Free gift on all orders above 100 eur”, and you may or may not to have this highlighted in the cart. :)

Reply:
@Hlavtox but auto applied cart rules are not getting highlighted in the cart anyway 🤔

Reply:
@kpodemski That's not true, they are highlighted if wanted.

![Snímek obrazovky 2024-02-27 085358](https://github.com/PrestaShop/PrestaShop/assets/6097524/84af9a9e-88c8-4225-9818-8ccdb2f97933)

Reply:
I tested gift + highlight on PrestaShop 8.1 and Classic and there was no information. I'll test a different rule.

Reply:
It must be a rule that is not in the cart already, limited by minimum amount for example. You can see that I tested it in the table. :-)